src/Tribe/Attendee_Registration/Main.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Attendee Registration core class
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Attendee_Registration__Main {
+	const QUERY_VAR = 'attendee-registration';
+	const DEFAULT_PAGE_SLUG = 'attendee-registration';
```

Comment:
Class constants like these are harder to deprecate than just standard class properties, so it's a good idea to [avoid them](https://github.com/moderntribe/the-events-calendar/pull/1861#discussion_r173348672) :)

Reply:
Thanks 👍